### ISSUE TITLE
make install doesn't work on MacOS

### ISSUE THREAD

**Issue by denismakogon** [hybridgroup/gocv#161]
`make install` fails on MacOS with the following error:
```
cd /tmp/opencv/opencv-3.4.1
/bin/sh: line 0: cd: /tmp/opencv/opencv-3.4.1: No such file or directory
make: *** [build] Error 1
```
because it downloads all files to current directory:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	opencv-3.4.1/
	opencv.zip
	opencv_contrib-3.4.1/
	opencv_contrib.zip

```
current directory is:
```
pwd
/Users/denismakogon/Documents/go/src/gocv.io/x/gocv
```

**Comment by deadprogram** [comment]
You should be using homebrew on macOS to install as discussed here:

https://github.com/hybridgroup/gocv/blob/master/README.md#macos

**Comment by deadprogram** [comment]
It does indeed have OpenCV Contrib, which is now installed as part of the default install for Homebrew:

https://github.com/Homebrew/homebrew-core/blob/master/Formula/opencv.rb#L29

**Comment by denismakogon** [comment]
Hm, okay.

Here's another error i get when trying to run tests locally:
```
go test -v ./...
# gocv.io/x/gocv
imgproc.cpp:27:26: error: expected '(' for function-style cast or type construction
imgproc.cpp:30:19: error: expected '(' for function-style cast or type construction
imgproc.cpp:133:28: error: expected '(' for function-style cast or type construction
FAIL	gocv.io/x/gocv [build failed]
# gocv.io/x/gocv
imgproc.cpp:27:26: error: expected '(' for function-style cast or type construction
imgproc.cpp:30:19: error: expected '(' for function-style cast or type construction
imgproc.cpp:133:28: error: expected '(' for function-style cast or type construction
FAIL	gocv.io/x/gocv/contrib [build failed]
FAIL	gocv.io/x/gocv/pvl [build failed]

```

**Comment by deadprogram** [comment]
You must also run `source env.sh` before running any GoCV code on macOS.

**Comment by denismakogon** [comment]
Doesn't much help there
```
/bin/bash ./env.sh
Brew install detected
Environment variables configured for OSX
Deniss-MBP-2:gocv denismakogon$ go test -v ./...
# gocv.io/x/gocv
imgproc.cpp:27:26: error: expected '(' for function-style cast or type construction
imgproc.cpp:30:19: error: expected '(' for function-style cast or type construction
imgproc.cpp:133:28: error: expected '(' for function-style cast or type construction
FAIL	gocv.io/x/gocv [build failed]
# gocv.io/x/gocv
imgproc.cpp:27:26: error: expected '(' for function-style cast or type construction
imgproc.cpp:30:19: error: expected '(' for function-style cast or type construction
imgproc.cpp:133:28: error: expected '(' for function-style cast or type construction
FAIL	gocv.io/x/gocv/contrib [build failed]
FAIL	gocv.io/x/gocv/pvl [build failed]
```

**Comment by deadprogram** [comment]
AH, you cannot run the `pvl` tests with the Intel CV SDK installed. Hence why the instructions to run the tests here: https://github.com/hybridgroup/gocv/blob/master/CONTRIBUTING.md#how-to-run-tests

**Comment by deadprogram** [comment]
The issue @denismakogon that you mention above within `imgproc.cpp` has been addressed in the `dev` branch. I will close this issue after the next release.

**Comment by deadprogram** [comment]
This code has now been released as part of v0.12.0 so now closing. Thanks again!
